drs_filer/errors/exceptions.py

```diff
@@ -0,0 +1,62 @@
+from connexion.exceptions import (
+    ExtraParameterProblem,
+    Forbidden,
+    Unauthorized,
+    BadRequestProblem
+)
+
+from werkzeug.exceptions import (
+    BadRequest,
+    BadGateway,
+    GatewayTimeout,
+    InternalServerError,
+    NotFound,
+    ServiceUnavailable,
+)
+
+exceptions = {
+    Exception: {
+        "title": "Internal Server Error",
+        "status": 500,
+    },
+    BadRequestProblem: {
```

Comment:
These should follow the error model from the specs, i.e., use fields `msg` and `status_code`, then you can use the error descriptions from the specs for the `msg` values, e.g. "The request is malformed" for `400` errors.